shared/actions/chat/creators.js

```diff
@@ -332,6 +336,10 @@ function threadLoadedOffline (conversationIDKey: Constants.ConversationIDKey): C
   return {payload: {conversationIDKey}, type: 'chat:threadLoadedOffline'}
 }
 
+function setSelectedRouteState (selectedConversation: Constants.ConversationIDKey, partialState: Object): SetRouteState {
```

Comment:
set state with an action instead of through the route tree helper (which we don't have access to right now)